src/components/GroupsPageComponents/components/GroupsPageTitle/hooks/useGroupsPageTitle.ts

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+import { SelectItem } from 'primereact/selectitem';
+
+export const GroupFilterOptions: SelectItem[] = [
+  { value: 'active', label: 'Только активные' },
+  { value: 'inactive', label: 'Только скрытые' },
+  { value: 'all', label: 'Все' },
+];
+
+export const useGroupsPageTitle = () => {
+  const [selectedOption, setSelectedOption] = useState<string>('active');
```

Comment:
Тут тоже. Еще момент: примитивы string, number, boolean можно не передавать в дженерики, так как их typescript автоматом определяет по переданному значению.
```suggestion
  const [selectedOption, setSelectedOption] = useState(GroupStatus.ACTIVE);
```